packages/apps/spaces/next.config.js

```diff
@@ -28,9 +29,9 @@ const config = {
       transform: '@spaces/hooks/*/{{ member }}',
     },
   },
-  optimization: {
-    mergeDuplicateChunks: true,
-  },
+  // optimization: {
```

Comment:
Why did you commented this code?

Reply:
There was a warning from next saying that `optimization` was not a known property for next.config. I think `optimization` was a property on an older version of webpack.